operations/iuf/stages/prepare_images.md

```diff
@@ -31,6 +32,117 @@ The following arguments are most often used with the `prepare-images` stage. See
 | Site variables                                | `-sv SITE_VARS`                  | Path to YAML file containing site defaults and any overrides                                          |
 | Recipe variables product mask                 | `-mrp MASK_RECIPE_PRODS`         | Mask the recipe variables file entries for the products specified, use product catalog values instead |
 
+## Artifacts created
+
+The artifacts created by the `prepare-images` stage can found by examining a Kubernetes ConfigMap associated with the IUF activity specified when executing the `prepare-images` stage. The following examples show how to find the
```

Comment:
Fix  --stage can found by--
The artifacts created by the prepare-images stage are found in the Kubernetes ConfigMap. Examine the activity ConfigMap associated with the IUF activity...etc.

Reply:
Fixed in next update.